src/template-parameters.json

```diff
@@ -272,6 +272,25 @@
         "help": "Set an initial user identifier applied to subsequent events.",
         "type": "TEXT",
         "simpleValueType": true
+      },
+      {
+        "displayName": "Use IIFE bundle",
+        "name": "useIIFE",
+        "help": "If your website has RequireJS, use IIFE bundle of search-insights to avoid any issue. (False by default)",
+        "type": "SELECT",
+        "macrosInSelect": true,
+        "simpleValueType": true,
```

Comment:
Does this mean that the default value would be set to `true` ? Or is it that the value defined elsewhere and I am misreading this?

Additionally, if it hasn't been done yet, it might be worth informing MathieuO regarding this change, since we might want to communicate on it a bit more.

Reply:
While I do not understand exactly what `simpleValueType` means, but I feel like it's true by default for primitive(simple?) values. On GTM UI, if I create a field with table type or something like that, that value disappears. Anyway, `Use IIEF` is `false` by default.

Pinging @mathieuolivier to see if you're aware of the context.

I have no concrete data, but I suspect Magento 2 + GTM hasn't work so far due to this issue.